test/unit/config/processEvent_test.js

```diff
@@ -0,0 +1,107 @@
+goog.module('measurementLibrary.testing.config.processEventConstructor');
+goog.setTestOnly();
+
+describe('After calling measure with `event` as the first parameter', () => {
+  let load;
+  let save;
+  let persistTime;
+  let processEvent;
+  let measure;
+  let dataLayer;
+
+  // Create a Mock Storage and Mock Processor object. Note that we cannot
+  // use jasmine.createSpyObject since it does not have a constructor.
+  // We need a class that will create an instance of itself that we can then
+  // access. Here this is done by making each method a spy.
+  class MockStorage {
+    constructor() {
+      this.load = load;
+      this.save = save;
+    }
+  }
+
+  class MockProcessor {
+    constructor() {
+      this.persistTime = persistTime;
+      this.processEvent = processEvent;
+    }
+    static getName() {
+      return 'processor';
+    }
+  }
+
+  beforeEach(() => {
+    load = jasmine.createSpy('load');
+    save = jasmine.createSpy('save');
+    persistTime = jasmine.createSpy('persistTime');
+    processEvent = jasmine.createSpy('processEvent');
+    // Reset the data layer we are using.
+    dataLayer = [];
+    measure = function() {
+      dataLayer.push(arguments);
+    };
+    setupMeasure(dataLayer);
+  });
+
+
+  describe('The call forwarded after a call to measure(`event`, ...)', () => {
+    /* TODO:(wolfblue@): Fix the issue here, it stems from data layer helper
```

Comment:
I would strongly prefer that we don't merge commented code, so please uncomment once https://github.com/google/data-layer-helper/pull/59 is merged and DLH is updated.

Reply:
It has been uncommented! I will save WIP code locally from now on